### ISSUE TITLE
Level: "Galton Box"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#104]
I think this is fairly easy, but I wanted to make a level that looks like the "Galton Box" game.  It's a school-fate-type game where you drop a penny in the top, which bounces off some nails before falling into one of several zones at the bottom.  If you get near one of the edges then you win a big prize, and if it falls near the middle you lose.  I found the name "Galton Box" by googling, but if anyone can think of a better name then I'm all ears.
```
:name=Galton Box
:description=
:num_rabbits=30
:num_to_save=21
:block=10
:climb=10
:dig=10
:bridge=10
:bash=10
:explode=10
:music=tryad-witness
####### O #######
######  #  ######
#####   *   #####
####  *   *  ####
###           ###
## Q*  *Q*  *Q ##
#               #
#################
:*=()pppppppppp
:*=(pppppppppp
:*=)pppppppppp
:*=(pppppppppp
:*=(pppppppppp
:*=)pppppppppp
:*=)pppppppppp
```

**Comment by andybalaam** [comment]
This looks really cool.  I'm going to be focussing on testing, publishing and waiting for feedback on release v0.6 but after that I'm looking forward to all these new levels!

**Comment by colonelfazackerley** [comment]
I have had a few tries at this, and failed. I am trying to get the code for retrieving levels from issues working. I will come back to this.

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
close me

**Comment by andybalaam** [comment]
I completed this using a dubious behaviour where you seem to be able to build a single block bridge near to a ceiling. I basically brute forced a bridge up from the far right directly to the exit. I am guessing there is a better way.

**Comment by andybalaam** [comment]
Backslash in hint 3.

**Comment by andybalaam** [comment]
I see I did it the same way as the hints. Was this the intention? I guess it's ok...

**Comment by colonelfazackerley** [comment]
It's the only way I could figure out. Hence the hints. When I did it, I got the minimum number out, which made me think it was the intended solution.

**Comment by tttppp** [comment]
Yep, that was the intended solution. There may be other solutions by bashing to drop bombs and bridging over them, but the idea was to 'sneak around' the bombs.

**Comment by andybalaam** [comment]
Cool.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721
